lcov.info

```diff
@@ -0,0 +1,18 @@
+mode: set
```

Comment:
What is the purpose of this file?

Reply:
sorry should not have been there. removed.